test/hw7/test_task1.py

```diff
@@ -0,0 +1,93 @@
+import pytest
+
+from All_home_works.hw7.task1 import find_occurrences
+
+
+@pytest.mark.parametrize(  # noqa: PT006
+    "input_data,element,expected",
+    [
+        ({"key1": "OK", "key2": "NOT OK"}, "OK", 1),
+        ({"key1": ["OK"], "key2": ["NOT OK"]}, "OK", 1),
+        ({"key1": ("OK",), "key2": ("NOT OK", "OK")}, "OK", 2),
+        ({"key1": {"OK"}, "key2": {"NOT OK"}}, "OK", 1),
+        ({"key1": {"key1": "OK"}, "key2": {"key1": "OK"}}, "OK", 2),
+        ({"key1": {"key1": ["OK"]}, "key2": {"key1": ("OK",)}}, "OK", 2),
+    ],
+)
+def test_find_occurrences__search_string(input_data, element, expected):
+    assert expected == find_occurrences(input_data, element)
+
+
+@pytest.mark.parametrize(  # noqa: PT006
+    "input_data,element,expected",
+    [
+        ({"key1": ["OK"], "key2": ["NOT OK"]}, ["OK"], 1),
+        ({"key1": ("OK",), "key2": ("NOT OK", "OK")}, ("OK",), 1),
+        ({"key1": {"OK"}, "key2": {"NOT OK"}}, {"OK"}, 1),
+        ({"key1": {"key1": ["OK"]}, "key2": {"key1": ("OK",)}}, ["OK"], 1),
+        (
+            {"key1": {"key1": ["OK"]}, "key2": {"key1": ("OK", {"key1": ["OK"]})}},
+            ["OK"],
+            2,
+        ),
+    ],
+)
+def test_find_occurrences__search__list__tuple__set(input_data, element, expected):
+    assert expected == find_occurrences(input_data, element)
+
+
+@pytest.mark.parametrize(  # noqa: PT006
```

Comment:
What is PT006 about?

Reply:
Thanks for asking. Maybe you will help me to sort out.
`@pytest.mark.parametrize("data,expected", [("a", "a"), ("b", "b")])`
GOT: PT006 wrong name(s) type expected tuple.

`@pytest.mark.parametrize("data,expected", (("a", "a"), ("b", "b")))`
GOT: PT007 wrong values expected list of tuples
AND: PT006 wrong name(s) type expected tuple

`@pytest.mark.parametrize("data,expected", (["a", "a"], ["b", "b"]))`
GOT: PT006 wrong name(s) type expected tuple, 
AND: PT007 wrong values type expected list of tuples